js/src/components/Dropdown/DropdownOption/index.js

```diff
@@ -56,11 +56,13 @@ export default class DropDownOption extends Component {
       activeClassName,
       disabledClassName,
       highlightedClassName,
+      value,
      } = this.props;
     return (
       <li
         className={classNames(
             'rdw-dropdownoption-default',
+            `rdw-dropdownoption-default-${value}`,
```

Comment:
Every dropdown item has a value prop that is unique to the item, so we can use that for specific css class.

Planning on targeting the css within our understory app, not here in the repo.